packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SubjectGroupViewer/components/SGVGridCell/SGVGridCell.js

```diff
@@ -54,6 +68,17 @@ function SGVGridCell (props) {
   const imageY = (cellHeight - imageHeight) / 2
 
   const clipPathID = `subjectGroupViewer-clipPath-${index}`
+  
```

Comment:
```suggestion
  const [checked, setChecked] = useState(cellAnnotated);
```
Initialise the `checked` state of each cell here.